controllers/datamovement_controller_rsync.go

```diff
@@ -284,15 +284,16 @@ func (r *DataMovementReconciler) monitorRsyncJob(ctx context.Context, dm *nnfv1a
 
 func (r *DataMovementReconciler) teardownRsyncJob(ctx context.Context, dm *nnfv1alpha1.NnfDataMovement) (*ctrl.Result, error) {
 	log := log.FromContext(ctx)
```

Comment:
Consider using the `WithName("...")` as you did in teardownLustreJob

Reply:
done